### PR TITLE
Basic UI for My Delegation

### PR DESCRIPTION
**Pull request by brandonfancher**
Available for now at the `/delegates/ui` path. This is all faked data...not even fixtures yet.

This set the stage for:
* Hooking up to delegation APIs and testing more extensively with fixture data
* Adding blank states (no election has happened yet, logged out, etc.)
* Move to the main `/delegates` path (or wherever its final resting place will be)

But those can be done in subsequent PRs.

### Screenshot
image

### CODE REVIEW

File: packages/webapp/src/members/components/temp-member-card-resign-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
```
Comment (cool-ant) [inline packages/webapp/src/members/components/temp-member-card-resign-modal.tsx:1]: Filename is "temp-"? Is that intentional?

File: packages/webapp/src/pages/delegates/ui/index.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                            />
+                        </Container>
+                    )}
+                </div>
+            ))}
+        </>
+    );
+};
+
+const levels: string[] = [
```
Comment (cool-ant) [inline packages/webapp/src/pages/delegates/ui/index.tsx:70]: We should either have a TODO comment here or make this dynamically determined.

File: packages/webapp/src/pages/election/components.tsx [hunk 3]
```diff
@@ -57,13 +57,17 @@ export const MembersPage = (props: Props) => {
...
                 {members.isLoading && "Loading members..."}
                 {members.error && "Fail to load members"}
             </Container>
             <MembersGrid members={members.data?.slice(1, 2)}>
                 {(member) => (
-                    <WinningMemberChip key={member.account} member={member} />
+                    <DelegateChip
+                        key={member.account}
+                        member={member}
+                        level="D3 - Chief Delegate"
```
Comment (cool-ant) [inline packages/webapp/src/pages/election/components.tsx:69]: Similar here w the hard coded value

File: packages/webapp/src/_app/ui/delegate-badge.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { FaStar } from "react-icons/fa";
+
+export const DelegateBadge = ({ size }: { size: number }) => (
```
Comment (sparkplug0025) [inline packages/webapp/src/_app/ui/delegate-badge.tsx:3]: Should we keep our size pattern? eg for buttons:
`export type ButtonSize = "sm" | "md" | "lg";`

File: packages/webapp/src/members/components/temp-member-card-resign-modal.tsx (hunk 1 above)
Comment (sparkplug0025) [reply packages/webapp/src/members/components/temp-member-card-resign-modal.tsx:1]: Probably also rename component from `MemberCard` to `MemberCardResignModal`

Comment (sparkplug0025) [reply packages/webapp/src/members/components/temp-member-card-resign-modal.tsx:1]: and update/remove the data-testid

File: packages/webapp/src/pages/delegates/ui/index.tsx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+            dehydratedState: dehydrate(queryClient),
+        },
+    };
+};
+
+interface Props {
+    delegatesPage: number;
+}
+
+const MEMBERS_PAGE_SIZE = 4;
```
Comment (sparkplug0025) [inline packages/webapp/src/pages/delegates/ui/index.tsx:23]: I don't think this is relevant for delegate. If it's related to the logged user we will not have the logged user session in our server, so hydrating it here does not make sense. (Unless we plan to implement user sessions handling in the server)

Comment (sparkplug0025) [reply packages/webapp/src/pages/delegates/ui/index.tsx:23]: tldr; this block can be removed and we can let the client execute the query as it's doing below

File: packages/webapp/src/members/components/temp-member-card-resign-modal.tsx (hunk 1 above)
Comment (brandonfancher) [reply packages/webapp/src/members/components/temp-member-card-resign-modal.tsx:1]: Yeah, this wasn't meant to me checked in. Sorry, guys!

File: packages/webapp/src/pages/delegates/ui/index.tsx (hunk 2 above)
Comment (brandonfancher) [reply packages/webapp/src/pages/delegates/ui/index.tsx:70]: I'll add a TODO. We'll make it dynamic when we hook up to the data.

File: packages/webapp/src/_app/ui/delegate-badge.tsx (hunk 4 above)
Comment (brandonfancher) [reply packages/webapp/src/_app/ui/delegate-badge.tsx:3]: I don't know...yet. For right now, I see this more like a composed react-icon where size is flexibly variable. But if we come up with a strong convention on badge use and sizing, then yes, I think we make this more opinionated.

File: packages/webapp/src/pages/delegates/ui/index.tsx (hunk 5 above)
Comment (brandonfancher) [reply packages/webapp/src/pages/delegates/ui/index.tsx:23]: I'll remove the boilerplate for now. I erred on the side of leaving stuff in for us to remove when we hook this UI up vs. making everything perfectly clean. We are going to need to decide what we want on this page when a member is not logged in. If, for example, we want to show the Head Chief / Satoshi on this page, that could potentially be SSR'd.

File: packages/webapp/src/members/components/temp-member-card-resign-modal.tsx (hunk 1 above)
Comment (brandonfancher) [reply packages/webapp/src/members/components/temp-member-card-resign-modal.tsx]: Removed.

File: packages/webapp/src/pages/election/components.tsx (hunk 3 above)
Comment (brandonfancher) [reply packages/webapp/src/pages/election/components.tsx:69]: Yes. This is just example UI for now. Not hooked up to any data. We'll make sure these are dynamic and are a matter of configuration. 👍

File: packages/webapp/src/pages/delegates/ui/index.tsx (hunk 2 above)
Comment (brandonfancher) [reply packages/webapp/src/pages/delegates/ui/index.tsx:59]: Done

**Review by cool-ant — APPROVED** [review]

**Pull request merged**